server/middleware/trial-requests.coffee

```diff
@@ -28,6 +30,14 @@ module.exports =
     trialRequest.set 'type', attrs.type
     database.validateDoc(trialRequest)
     trialRequest = yield trialRequest.save()
+    if trialRequest.get('properties')?.referredBy is 'sunburst'
+      context =
+        email_id: sendwithus.templates.sunburst_referrral
+        recipient:
+          address: config.sunburst.email
+        email_data:
+          trial_request: JSON.stringify(trialRequest.toJSON(), null, 2)
+      sendwithus.api.send context, _.noop
```

Comment:
*Ideally* this gets a test [here](https://github.com/codecombat/codecombat/blob/master/spec/server/functional/trial_request.spec.coffee#L22). Imagine one just like the first one, with the `referredBy` set to 'sunburst', spying on `sendwithus.api.send` and checking the context of the call. Not critical, but since this is a piece that we won't likely notice if we break it, could be helpful.